### PR TITLE
prevent jumping to top of page when modal opens

### PR DESCRIPTION
**Pull request by abemassry**
Fixes a problem where if a modal is opened at the bottom of the page, it cause a scroll to the top of the page.

### CODE REVIEW

File: components/scss/components/modals/Modal.scss [hunk 1]
```diff
@@ -69,3 +69,7 @@
         font-weight: bold;
     }
 }
+
+.layout--modal {
+    overflow: visible;
```
Comment (eatonphil) [inline components/scss/components/modals/Modal.scss]: Look for the rule that explicitly sets .layout--modal to `overflow-y: hidden` and change it there instead.

**Comment by eatonphil** [comment]
The reason I set the rule originally to be overflow-y hidden was so that the background didn't scroll behind a modal. idk which is worse, probably how it was

File: components/scss/components/modals/Modal.scss (hunk 1 above)
Comment (mparke) [inline components/scss/components/modals/Modal.scss]: this is the only .layout--modal rule

Comment (eatonphil) [inline components/scss/components/modals/Modal.scss]: https://github.com/linode/manager/blob/master/scss/layout/Layout.scss#L13

Comment (mparke) [inline components/scss/components/modals/Modal.scss]: thanks

**Pull request merged**